usage/conf/routes

```diff
@@ -1,5 +1,7 @@
 GET     /                                               controllers.UsageApi.index
 
+GET     /usage/media/:mediaId                           controllers.UsageApi.forMedia(mediaId: String)
```

Comment:
Guess I wasn't thinking about presenting a list of all usages on `/usage` so don't care about pluralising in the REST style. But come to think of it we might actually might do that so it's probably worth it.